example/ios/Podfile

```diff
@@ -25,6 +31,7 @@ target 'example' do
   post_install do |installer|
     installer.pods_project.targets.each do |target|
       target.build_configurations.each do |config|
+        config.build_settings['SWIFT_VERSION'] = '5.0'
         deployment_target = config.build_settings['IPHONEOS_DEPLOYMENT_TARGET']
```

Comment:
Here, we are converting the pods to install using swift 5. I hadn't thought about this issue before, its another area where need to get the ios SDK updated to swift 6.